### PR TITLE
fix: Update resource requirements for Eclipse Che Operator, Operands, …

### PR DESCRIPTION
**Pull request by ibuziuk**
…and Dev Workspace Controller

## What does this pull request change?
Update Table 3.

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-5085

## Specify the version of the product this pull request applies to
Merge to `main`
Cherry-pick to `7.60`, `7.61`, `7.62`

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 1]
```diff
@@ -88,10 +88,10 @@ When this section is empty, {prod-short} only loads the implicit components.
 
 . Sum up the resources required for each workspace, and multiply them by the running workspaces count.
 
-. Sum up the server components requirements.
+. Sum up the requirements for the {prod-short} Operator, Operands, and {devworkspace} Controller
 +
 [cols="1a,1a,1a,1a,1a,1a,1a",options="header"]
-.{prod-short} server components default requirements
+. Default requirements {prod-short} Operator, Operands, and {devworkspace} Controller
```
Comment (max-cx) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]:
```suggestion
.Default requirements {prod-short} Operator, Operands, and {devworkspace} Controller
```

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 2]
```diff
@@ -88,10 +88,10 @@ When this section is empty, {prod-short} only loads the implicit components.
 
 . Sum up the resources required for each workspace, and multiply them by the running workspaces count.
 
-. Sum up the server components requirements.
+. Sum up the requirements for the {prod-short} Operator, Operands, and {devworkspace} Controller
 +
 [cols="1a,1a,1a,1a,1a,1a,1a",options="header"]
-.{prod-short} server components default requirements
+.Default requirements {prod-short} Operator, Operands, and {devworkspace} Controller
```
Comment (max-cx) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]:
```suggestion
.Default requirements for the {prod-short} Operator, Operands, and {devworkspace} Controller
```

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 3]
```diff
@@ -222,9 +222,9 @@ When this section is empty, {prod-short} only loads the implicit components.
 |N/A
 
 3+>s|Total
->s|8.5 GiB
->s|1.1 GiB
+>s|9 GiB
```
Comment (max-cx) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc:225]:
```suggestion
>s|8.9 GiB
```

Comment (ibuziuk) [reply modules/administration-guide/pages/calculating-che-resource-requirements.adoc:225]: it is 9048 according to my calculations, no?

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 4]
```diff
@@ -178,7 +178,7 @@ When this section is empty, {prod-short} only loads the implicit components.
 |`{devworkspace-id}-controller`
 |1 GiB
 |100 MiB
-|1
+|1000m
```
Comment (max-cx) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]:
```suggestion
|1
```

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc (hunk 3 above)
Comment (max-cx) [reply modules/administration-guide/pages/calculating-che-resource-requirements.adoc:225]: Memory limit: 256+1000+256+4000+256+512+512+256+256+1000+300+300=8904=8.9 GiB
Memory request: 64+512+32+128+64+64+64+32+32+100+20+50+20=1182=1.2 GiB
CPU limit: 500+1000+500+1000+500+500+500+500+500+1000+200+200=6900=6.9
CPU request: 100+100+100+100+50+100+100+100+100+250+100+10+100=1310=1.3

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 5]
```diff
@@ -178,7 +178,7 @@ When this section is empty, {prod-short} only loads the implicit components.
 |`{devworkspace-id}-controller`
 |1 GiB
 |100 MiB
-|1
+|1000 m
```
Comment (max-cx) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]: There have been three 1's in that column, so I suggest avoiding a state where we have two 1's and one 1000m . That's why I suggest reverting this:
```suggestion
|1
```

**Review by max-cx — APPROVED** [review]
Approving with one reservation regarding the original content that needs to be fixed in another PR:

`*` should not be there:

https://github.com/eclipse-che/che-docs/pull/2576/files#diff-fff979ead8d9223070416bece5eb5e7fac9493bbcab9860371b6df6b6a8bf137R122

image

**Pull request merged**